services/movie.py

```diff
@@ -0,0 +1,32 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> list[Movie]:
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(genres__id__in=genres_ids,
+                                    actors__id__in=actors_ids)
```

Comment:
You can remove that. Just not return a querysets in the next conditions, just filter them there